graphene/types/mutation.py

```diff
@@ -22,7 +22,44 @@ class MutationOptions(ObjectTypeOptions):
 
 class Mutation(ObjectType):
     """
-    Mutation Type Definition
+    Object Type Definition (mutation field)
+
+    Mutation is a convenience type that helps us build a Field which takes Arguments and returns a
+    mutation Output ObjectType.
+
+    .. code:: python
+
+        class CreatePerson(graphene.Mutation):
+            class Arguments:
+                name = graphene.String()
+
+            ok = graphene.Boolean()
+            person = graphene.Field(Person)
+
+            def mutate(parent, info, name):
+                person = Person(name=name)
+                ok = True
+                return CreatePerson(person=person, ok=ok)
+
+        class Mutation(graphene.ObjectType):
+            create_person = CreatePerson.Field()
+
+    Meta class options (optional):
+        output (graphene.ObjectType): Or ``Output`` inner class with attributes on Mutation class.
+            Or attributes from Mutation class. Fields which can be returned from this mutation
+            field.
+        resolver (Callable resolver method): Or ``mutate`` method on Mutation class. Perform data
+            change and return output.
+        arguments (Dict[str, graphene.Argument]): Or ``Arguments`` inner class with attributes on
+            Mutation class. Arguments to use for the mutation Field.
+        name (str): the name of the GraphQL type (must be unique in schema). Defaults to class
+            name.
+        description (str): the description of the GraphQL type in the schema. Defaults to class
+            docstring.
+        interfaces (Iterable[graphene.Interface]): GraphQL interfaces to extend with the payload
```

Comment:
I don't think this is an option on `Mutation`

Reply:
It actually is! `MutationOptions` inherits from `ObjectTypeOptions`, so it has the capability to implement interfaces (Union should be valid for Mutation Payload as well).

I believe the idea of a Mutation being a different concept from an ObjectType is unique to graphene. Mutation is a convenience that creates an ObjectType (MutationPayload) with arguments (MutationArguments or MutationInput). In core JS GraphQL, I believe you just define field, argument, objects and resolvers for mutations, same as for a query. I think you can even do this with Graphene, but I haven't tried it yet.

Reply:
Huh you're completely right. I had no idea. I just tried it in a project I work on and it doesn't seem to have any effect though. Do you think it's worth documenting here if it doesn't have any effect? It could be quite misleading.

Reply:
Oh really? This might be a bug or omission in the mutation code.. :thinking: 

I found a couple of things like this while documenting the API. I'll update the docs to indicate that this argument is accepted, but not implemented and raise an issue to track that.

Reply:
Thanks!